xml/System.ServiceModel.Syndication/SyndicationTextInput.xml

```diff
@@ -16,8 +16,36 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a "textInput" entry from an RSS syndication feed.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The RSS specification allows for a 'textInput' item to be included within a syndication feed that the publisher can userinput
```

Comment:
Should the RSS spec be linked to? There's a deep link to this specific section of the spec: https://cyber.harvard.edu/rss/rss.html#lttextinputgtSubelementOfLtchannelgt